docs/topics/http/file-uploads.txt

```diff
@@ -38,9 +38,15 @@ subclass) in the form. So the data from the above form would
 be accessible as ``request.FILES['file']``.
 
 Note that :attr:`request.FILES <django.http.HttpRequest.FILES>` will only
-contain data if the request method was ``POST`` and the ``<form>`` that posted
-the request has the attribute ``enctype="multipart/form-data"``. Otherwise,
-``request.FILES`` will be empty.
+contain data from form fields if the request method was ``POST`` and
+the ``<form>`` that posted the request has the attribute
+``enctype="multipart/form-data"`` and contains at least one
+``<intput type="file">`` field.
+Or more technically if request method was ``POST``, the
+``Content-Type`` general header was ``multipart/form-data`` and
+the ``Content-Disposition`` subpart header contains the filename parameter
+(default behaviour in browsers for this ``enctype``).
+Otherwise, ``request.FILES`` will be empty.
```

Comment:
I think I might trim it down to maybe just this: 

```suggestion
contain data from form fields if the request method was ``POST``, at least one
file field was actually posted, and the ``<form>`` that posted the request has
the attribute ``enctype="multipart/form-data"``.
```

I think, in this context, the whole _more technically_ bit is just too much.

Reply:
I'm fine with it, it's definitely more precise than the current version.
Thanks for your help and time!

Cheers